scania-digital-design-system/sdds
# chore: change theme-vars style files name

**Describe pull-request**  
Changed the files with mode variables from components-theme-vars to components-vars

**Solving issue**  
Fixes: [AB#3478](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/3478)

**How to test**  
_Add description how to test if possible_
1. Go to storybook and check that all components still looks like they should

**Suggested test steps**
